satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -73,244 +73,274 @@ datasets:
+    standard_name: air_pressure_at_cloud_top_in_lower_atmosphere_layer
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 38
     units: Pa
-    standard_name: air_pressure_at_cloud_top
 
   error_in_lower_layer_cloud_optical_depth:
     name: error_in_lower_layer_cloud_optical_depth
+    long_name: Lower Cloud Optical Depth Error Estimate
+    standard_name: atmosphere_optical_thickness_due_to_cloud_in_lower_atmosphere_layer standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 39
     units: "1"
-    long_name: cloud_optical_depth_standard_error
 
   error_in_lower_layer_cloud_top_pressure:
     name: error_in_lower_layer_cloud_top_pressure
+    long_name: Lower Cloud Top Pressure Error Estimate
+    standard_name: air_pressure_at_cloud_top_in_lower_atmosphere_layer standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 40
     units: Pa
-    standard_name: air_pressure_at_cloud_top_standard_error
 
   fire_probability:
     name: fire_probability
+    long_name: Fire Probability
+    standard_name: fire_probability
     resolution: 3000.403165817
     file_type: grib_seviri_fir
     parameter_number: 192
     units: "%"
-    long_name: fire_probability
 
   active_fires:
     name: active_fires
+    long_name: Active Fire Classification
+    standard_name: active_fire_classification
     resolution: 3000.403165817
     file_type: grib_seviri_fir
     parameter_number: 9
     units: "1"
-    long_name: active_fire_classification
 
   aerosol_optical_thickness_vis06:
     name: aerosol_optical_thickness_vis06
+    long_name: Aerosol optical Thickness at 0.6um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 20
     units: "1"
-    long_name: aerosol_optical_thickness_vis06
 
   aerosol_optical_thickness_vis08:
     name: aerosol_optical_thickness_vis08
+    long_name: Aerosol optical Thickness at 0.8um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 21
     units: "1"
-    long_name: aerosol_optical_thickness_vis08
 
   aerosol_optical_thickness_vis16:
     name: aerosol_optical_thickness_vis16
+    long_name: Aerosol optical Thickness at 1.6um
+    standard_name: atmosphere_absorption_optical_thickness_due_to_ambient_aerosol
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 22
     units: "1"
-    long_name: aerosol_optical_thickness_vis16
 
   angstroem_coefficient:
     name: angstroem_coefficient
+    long_name: Angstroem Coefficient
+    standard_name: aerosol_angstrom_exponent
     resolution: 9001.209497451
     file_type: grib_seviri_aes
     parameter_number: 23
     units: "1"
-    long_name: angstroem_coefficient
 
   aes_quality:
```

Comment:
Please add `flag_values` and `flag_meanings`.

Reply:
The AES quality is actually not an enumerated flag.
It's a value between 0 and 100 derived using the scene quality, but not as written in the ASD (it's more like the average of the inverse of the scene quality)

Reply:
Seems like you've added the `flag_values` and `flag_meanings` from the cloud mask here, which is probably a mistake given your response above?